Python TP/func_test_config.txt
1 1 3 1 1 1 1 1	1	1	1	3
2 1 6 0 1 0 1 0	1	1	1	6
3 1 0 0 0 0 0 0	0	0	1	0
4 1 0 0 0 0 0 0	0	0	1	0
5	1 7 0 0 0	0 0	1	1	1	7	


Python TP/configuration_IEEE.txt
1          1          1          0         1        1          1        1          1       1       1       1 
2          1          1          0         1        0          1        0          1       1       1       1
3          1          2          1         1        1          1        1          1       1       1       2
4          1          5          1         1        1          1        1          1       1       1       5
5          1          3          1         1        1          1        1          1       1       1       3
6          2          2          0         1        0          1        0          1       1       2       2
7          2          2          0         1        0          1        0          1       1       2       2
8          2          4          2         1        1          1        1          1       1       2       4
9          2          6          2         1        1          1        1          1       1       2       6
10         3          9          3         1        1          1        1          1       1       3       9
11         3          3          0         1        0          1        0          1       1       3       3
12         3          24         3         1        1          1        1          1       1       3       24
13         4          9          4         1        1          1        1          1       1       4       9 
14         4          4          0         1        0          1        0          1       1       4       4 
15         5          10         5         1        1          1        1          1       1       5       10 
16         5          5          0         1        0          1        0          1       1       5       5 
17         6          6          0         1        0          1        0          1       1       6       6 
18         6          10         6         1        1          1        1          1       1       6       10 
19         7          7          0         1        0          1        0          1       1       7       7 
20         7          8          7         1        1          1        1          1       1       7       8 
21         7          7          0         1        0          1        0          1       1       7       7 
22         8          8          0         1        0          1        0          1       1       8       8 
23         8          9          8         1        1          1        1          1       1       8       9 
24         8          10         8         1        1          1        1          1       1       8       10 
25         9          12         9         1        1          1        1          1       1       9       12 
26         9          11         9         1        1          1        1          1       1       9       11 
27         9          9          0         1        0          1        0          1       1       9       9 
28         10         11         10        1        1          1        1          1       1       10      11 
29         10         12         10        1        1          1        1          1       1       10      12 
30         10         10         0         1        0          1        0          1       1       10      10 
31         11         13         11        1        1          1        1          1       1       11      13 
32         11         14         11        1        1          1        1          1       1       11      14
33         12         23         12        1        1          1        1          1       1       12      23
34         12         13         12        1        1          1        1          1       1       12      13
35         13         13         0         1        0          1        0          1       1       13      13
36         13         23         13        1        1          1        1          1       1       13      23
37         13         13         0         1        0          1        0          1       1       13      13
38         14         14         0         1        0          1        0          1       1       14      14
39         14         16         14        1        1          1        1          1       1       14      16
40         14         14         0         1        0          1        0          1       1       14      14
41         15         15         0         1        0          1        0          1       1       15      15 
42         15         24         15        1        1          1        1          1       1       15      24 
43         15         16         15        1        1          1        1          1       1       15      16 
44         15         15         0         1        0          1        0          1       1       15      15
45         15         21         15        1        1          1        1          1       1       15      21
46         15         21         15        1        1          1        1          1       1       15      21
47         16         16         0         1        0          1        0          1       1       16      16
48         16         16         0         1        0          1        0          1       1       16      16
49         16         17         16        1        1          1        1          1       1       16      17
50         16         19         16        1        1          1        1          1       1       16      19
51         17         18         17        1        1          1        1          1       1       17      18
52         17         22         17        1        1          1        1          1       1       17      22
53         18         18         0         1        0          1        0          1       1       18      18
54         18         18         0         1        0          1        0          1       1       18      18
55         18         21         18        1        1          1        1          1       1       18      21
56         18         21         18        1        1          1        1          1       1       18      21
57         19         20         19        1        1          1        1          1       1       19      20
58         19         20         19        1        1          1        1          1       1       19      20
59         19         19         0         1        0          1        0          1       1       19      19
60         20         20         0         1        0          1        0          1       1       20      20
61         20         23         20        1        1          1        1          1       1       20      23
62         20         23         20        1        1          1        1          1       1       20      23
63         21         21         0         1        0          1        0          1       1       21      21
64         21         22         21        1        1          1        1          1       1       21      22
65         22         22         0         1        0          1        0          1       1       22      22
66         23         23         0         1        0          1        0          1       1       23      23
67         1          0          0         0        0          0        0          0       0       1       0 
68         2          0          0         0        0          0        0          0       0       1       0
69         3          0          0         0        0          0        0          0       0       1       0
70         4          0          0         0        0          0        0          0       0       1       0
71         5          0          0         0        0          0        0          0       0       1       0
72         6          0          0         0        0          0        0          0       0       1       0
73         7          0          0         0        0          0        0          0       0       1       0
74         8          0          0         0        0          0        0          0       0       1       0
75         9          0          0         0        0          0        0          0       0       1       0
76         10         0          0         0        0          0        0          0       0       1       0
77         11         0          0         0        0          0        0          0       0       1       0
78         12         0          0         0        0          0        0          0       0       1       0
79         13         0          0         0        0          0        0          0       0       1       0
80         14         0          0         0        0          0        0          0       0       1       0
81         15         0          0         0        0          0        0          0       0       1       0
82         15         0          0         0        0          0        0          0       0       1       0
83         16         0          0         0        0          0        0          0       0       1       0
84         17         0          0         0        0          0        0          0       0       1       0
85         18         0          0         0        0          0        0          0       0       1       0
86         19         0          0         0        0          0        0          0       0       1       0
87         20         0          0         0        0          0        0          0       0       1       0
88         21         0          0         0        0          0        0          0       0       1       0
89         22         0          0         0        0          0        0          0       0       1       0
90         24         0          0         0        0          0        0          0       0       1       0

MATLAB/configuration.m
          %Circuit.Num    From        To       From     Status    Status    Lf.Ava    Lf.Ava     Area     Area    O.F     O.T
          Conf=[1          1          1          0         1        1          1        1          1       1       1       1 
                2          1          1          0         1        0          1        0          1       1       1       1
                3          1          2          1         1        1          1        1          1       1       1       2
                4          1          5          1         1        1          1        1          1       1       1       5
                5          1          3          1         1        1          1        1          1       1       1       3
                6          2          2          0         1        0          1        0          1       1       2       2
                7          2          2          0         1        0          1        0          1       1       2       2
                8          2          4          2         1        1          1        1          1       1       2       4
                9          2          6          2         1        1          1        1          1       1       2       6
                10         3          9          3         1        1          1        1          1       1       3       9
                11         3          3          0         1        0          1        0          1       1       3       3
                12         3          24         3         1        1          1        1          1       1       3       24
                13         4          9          4         1        1          1        1          1       1       4       9 
                14         4          4          0         1        0          1        0          1       1       4       4 
                15         5          10         5         1        1          1        1          1       1       5       10 
                16         5          5          0         1        0          1        0          1       1       5       5 
                17         6          6          0         1        0          1        0          1       1       6       6 
                18         6          10         6         1        1          1        1          1       1       6       10 
                19         7          7          0         1        0          1        0          1       1       7       7 
                20         7          8          7         1        1          1        1          1       1       7       8 
                21         7          7          0         1        0          1        0          1       1       7       7 
                22         8          8          0         1        0          1        0          1       1       8       8 
                23         8          9          8         1        1          1        1          1       1       8       9 
                24         8          10         8         1        1          1        1          1       1       8       10 
                25         9          12         9         1        1          1        1          1       1       9       12 
                26         9          11         9         1        1          1        1          1       1       9       11 
                27         9          9          0         1        0          1        0          1       1       9       9 
                28         10         11         10        1        1          1        1          1       1       10      11 
                29         10         12         10        1        1          1        1          1       1       10      12 
                30         10         10         0         1        0          1        0          1       1       10      10 
                31         11         13         11        1        1          1        1          1       1       11      13 
                32         11         14         11        1        1          1        1          1       1       11      14
                33         12         23         12        1        1          1        1          1       1       12      23
                34         12         13         12        1        1          1        1          1       1       12      13
                35         13         13         0         1        0          1        0          1       1       13      13
                36         13         23         13        1        1          1        1          1       1       13      23
                37         13         13         0         1        0          1        0          1       1       13      13
                38         14         14         0         1        0          1        0          1       1       14      14
                39         14         16         14        1        1          1        1          1       1       14      16
                40         14         14         0         1        0          1        0          1       1       14      14
                41         15         15         0         1        0          1        0          1       1       15      15 
                42         15         24         15        1        1          1        1          1       1       15      24 
                43         15         16         15        1        1          1        1          1       1       15      16 
                44         15         15         0         1        0          1        0          1       1       15      15
                45         15         21         15        1        1          1        1          1       1       15      21
                46         15         21         15        1        1          1        1          1       1       15      21
                47         16         16         0         1        0          1        0          1       1       16      16
                48         16         16         0         1        0          1        0          1       1       16      16
                49         16         17         16        1        1          1        1          1       1       16      17
                50         16         19         16        1        1          1        1          1       1       16      19
                51         17         18         17        1        1          1        1          1       1       17      18
                52         17         22         17        1        1          1        1          1       1       17      22
                53         18         18         0         1        0          1        0          1       1       18      18
                54         18         18         0         1        0          1        0          1       1       18      18
                55         18         21         18        1        1          1        1          1       1       18      21
                56         18         21         18        1        1          1        1          1       1       18      21
                57         19         20         19        1        1          1        1          1       1       19      20
                58         19         20         19        1        1          1        1          1       1       19      20
                59         19         19         0         1        0          1        0          1       1       19      19
                60         20         20         0         1        0          1        0          1       1       20      20
                61         20         23         20        1        1          1        1          1       1       20      23
                62         20         23         20        1        1          1        1          1       1       20      23
                63         21         21         0         1        0          1        0          1       1       21      21
                64         21         22         21        1        1          1        1          1       1       21      22
                65         22         22         0         1        0          1        0          1       1       22      22
                66         23         23         0         1        0          1        0          1       1       23      23
                67         1          0          0         0        0          0        0          0       0       1       0 
                68         2          0          0         0        0          0        0          0       0       1       0
                69         3          0          0         0        0          0        0          0       0       1       0
                70         4          0          0         0        0          0        0          0       0       1       0
                71         5          0          0         0        0          0        0          0       0       1       0
                72         6          0          0         0        0          0        0          0       0       1       0
                73         7          0          0         0        0          0        0          0       0       1       0
                74         8          0          0         0        0          0        0          0       0       1       0
                75         9          0          0         0        0          0        0          0       0       1       0
                76         10         0          0         0        0          0        0          0       0       1       0
                77         11         0          0         0        0          0        0          0       0       1       0
                78         12         0          0         0        0          0        0          0       0       1       0
                79         13         0          0         0        0          0        0          0       0       1       0
                80         14         0          0         0        0          0        0          0       0       1       0
                81         15         0          0         0        0          0        0          0       0       1       0
                82         15         0          0         0        0          0        0          0       0       1       0
                83         16         0          0         0        0          0        0          0       0       1       0
                84         17         0          0         0        0          0        0          0       0       1       0
                85         18         0          0         0        0          0        0          0       0       1       0
                86         19         0          0         0        0          0        0          0       0       1       0
                87         20         0          0         0        0          0        0          0       0       1       0
                88         21         0          0         0        0          0        0          0       0       1       0
                89         22         0          0         0        0          0        0          0       0       1       0
                90         24         0          0         0        0          0        0          0       0       1       0];
                